kits/survey/contracts/SurveyKit.sol

```diff
@@ -22,8 +22,10 @@ contract SurveyKit is APMNamehash, KitBase {
 
     event DeployInstance(address dao, address indexed token);
 
-    // factory must be set up w/o EVMScript support
-    constructor(DAOFactory _fac, ENS _ens) KitBase(_fac, _ens) public {}
+    constructor(DAOFactory _fac, ENS _ens) KitBase(_fac, _ens) public {
+        // factory must be set up w/o EVMScript support
+        //require(address(_fac.regFactory()) == address(0));
```

Comment:
Should this be uncommented?

Reply:
Oops!! :open_mouth: I was testing if it was working without it and forgot to put it back. Thanks!!